dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcTestDataCreatorService.java

```diff
@@ -92,10 +95,19 @@ public Set<Integer> createFastTrackTestData(
   }
 
   private void processNegativeTests(
-          final FdcTestType testType, final Integer repOrderId, final Integer fdcId, final int monthsAfterSysDate) {
+          final FdcTestType testType, final Integer repOrderId, final Integer fdcId, final PickupType pickupType) {
     switch (testType) {
-      case NEGATIVE_SOD -> testDataClient.updateRepOrderSentenceOrderDate(UpdateRepOrder.builder()
-              .repId(repOrderId).sentenceOrderDate(LocalDate.now().plusMonths(monthsAfterSysDate)).build());
+      case NEGATIVE_SOD -> {
+        switch (pickupType) {
+          case DELAYED_PICKUP -> {
+            Map<String, Object> repOrderWithNullSOD = new HashMap<>();
+            repOrderWithNullSOD.put("sentenceOrderDate", null);
+            testDataClient.updateRepOrderSentenceOrderDateToNull(repOrderId, repOrderWithNullSOD);
+          }
+          case FAST_TRACK -> testDataClient.updateRepOrderSentenceOrderDate(UpdateRepOrder.builder()
+              .repId(repOrderId).sentenceOrderDate(LocalDate.now().plusMonths(-7)).build());
```

Comment:
I guess it doesn't matter, so not going to fail the review, but could use `minusMonths(7)` here: might be clearer than `plusMonths(-7)`.

Reply:
Thanks, changed. This was a hangover from previous functionality where sometimes +ve months were to be added.